h2o-mapreduce-generic/build.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'java'
+
+description = 'H2O Generic MapReduce Driver for Hadoop'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+compileJava {
+  options.debug = true
+}
+
+dependencies {
+  // we need to compile against something - the versions here are not important - assemblies will bundle the correct versions
+  compileOnly "org.apache.hive:hive-jdbc:0.13.1-cdh5.2.0"
+  compile "org.apache.hadoop:hadoop-client:2.5.0-cdh5.2.0"
```

Comment:
1. yeah, could be made `compileOnly` but it doesn't really matter - all the dependencies are excluded in the assembly
2. yep, it works because the hadoop-3 assemblies have the correct dependency added

I can change it to compileOnly - I have no strict preference. This artifact is not even deployed to Maven so no 3rd party will most likely ever depend on it.

Reply:
I was worried these classes will be missing at runtime. But if we add them, then it is ok. Tests obviously pass, so it is ok.